api/socketIO.js

```diff
@@ -15,14 +15,15 @@ if (redisConfig) {
   const pubClient = redis.createClient(redisConfig);
   const subClient = redis.createClient(redisConfig);
 
-  socketIO.adapter(redisAdapter({ pubClient, subClient }));
-
   pubClient.on('error', (err) => {
     logger.error(`redisAdapter pubClient error: ${err}`);
   });
   subClient.on('error', (err) => {
     logger.error(`redisAdapter subClient error: ${err}`);
   });
+
+  socketIO.adapter(redisAdapter({ pubClient, subClient }));
```

Comment:
does moving this have an effect?

Reply:
not sure ... i was just assigning the error handling before attaching the client to the adapter

Reply:
it appaers socket.use is no longer used ... but io.use still appears to be used ... in this io is socketIO